client/src/app/historyFolder/ViewHistory.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from "react";
+import "./history.css";
+// Post Card Component
+const PostCard: React.FC<{ index: number }> = ({ index }) => {
+  const [likes, setLikes] = useState(0); // save for when changing likes
+
+  return (
+    <div
+      style={{
+        width: 297,
+        height: 468,
```

Comment:
Consider using relative units for cleaner, more maintainable, and optimized code.